internal/runners/clean/stop.go

```diff
@@ -55,7 +56,11 @@ func stopServices(cfg configurable, out output.Outputer, ignoreErrors bool) erro
 		var isStopped bool
 		m := svcmanager.New(cfg)
 		for x := 0; x < 30; x++ {
-			isStopped = !m.Ready()
+			err = m.Ready()
+			if err != nil {
+				logging.Debug("Ready error: %v", err)
+				isStopped = true
+			}
```

Comment:
Please use a separate bool for the ready check itself. The Ready() function can then decide that error equals "not ready".